services/docgen-api/index.js

```diff
@@ -8,7 +8,7 @@ const express = require(`express`);
 const bodyParser = require(`body-parser`);
 const crypto = require("crypto");
 
-const templatedir = "/tmp/tmp-reports/";
+const templatedir = "/app/templates/";
```

Comment:
I don't think this is correct, was updated to /tmp/tmp-reports in the other PR

Reply:
Nevermind, see that you changed.

Reply:
fixed.